Sources/FluentPostgreSQL/FluentPostgreSQLProvider.swift

```diff
@@ -25,5 +26,39 @@ public final class FluentPostgreSQLProvider: Provider {
     public func boot(_ worker: Container) throws { }
 }
 
+/// check server version and override _globalEnableIdentityColumns
+/// dirty workaround for missing pre migration hook 
+public final class PostgreSQLVersionCheckProvider: Provider {
+    /// See `Provider.repositoryName`
+    public static let repositoryName = "fluent-postgresql-versioncheck"
```

Comment:
this should be `fluent-postgresql` (name of the gh repo)

Reply:
Done. 
I didn't know why this used, I thougt some directory identifier.